app/.storybook/preview.js

```diff
@@ -23,6 +23,22 @@ const parameters = {
   },
   // Configure i18next and locale/dropdown options.
   i18n,
+  options: {
+    storySort: {
+      method: "alphabetical",
```

Comment:
question: what happens by default? is it just a random order?

Reply:
I actually don't know what this means, but this is what they say the default is:

> Out of the box, Storybook sorts stories based on the order in which they are imported.

https://storybook.js.org/docs/react/writing-stories/naming-components-and-hierarchy#sorting-stories